xml/networking-dcn_core.xml

```diff
@@ -242,8 +241,7 @@
   </listitem>
   <listitem>
    <para>
-    Set up the third-party area as you would when deploying your cloud with the
-    DCN ml2 mechanism driver.
+    Set up the third-party area.
```

Comment:
This makes no sense anymore -- you need to copy the section on setting up the third-party area from deleted file here... I would guess it starts a line 819, no idea where it ends :(. Again, a technical expert might be helpful.